Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2056,11 +2021,11 @@ extension BrowserViewController: TabDelegate {
 extension BrowserViewController: SearchViewControllerDelegate {
     func searchViewController(_ searchViewController: SearchViewController, didSubmit query: String) {
         topToolbar.leaveOverlayMode()
-        processAddressBar(text: query)
+        processAddressBar(text: query, visitType: .typed)
     }
     
     func searchViewController(_ searchViewController: SearchViewController, didSelectURL url: URL) {
-        finishEditingAndSubmit(url, isBookmark: false)
+        finishEditingAndSubmit(url, visitType: .typed)
```

Comment:
A selected URL from suggestions or QRCode is considered `typed`?

Reply:
Selected URL from suggestions is always considered typed URL before we removed VisitType from the app. but the QRCode is a great point I didnt think about or consider.

However IMO the QR code should be also considered as typed since the user is scanning this code and making app open a URL. I feel like this is something we should be syncing. Please let me know If you think differently.